components/cdc/src/delegate.rs

```diff
@@ -596,6 +601,35 @@ impl Delegate {
                         continue;
                     }
 
+                    if self.txn_extra_op == TxnExtraOp::ReadOldValue {
+                        old_value_reader = old_value_reader.or_else(|| {
+                            // Hack: init a empty region to pass the check
+                            let mut region = Region::default();
+                            region.mut_peers().push(Peer::default());
+                            let snapshot = RegionSnapshot::from_snapshot(
```

Comment:
Can we take a snapshot in raftstore and pass it to CDC?

Reply:
🤔 We could do this after CDC was abstracted in engine traits.